lib/api/onnx.ts

```diff
@@ -14,7 +14,16 @@ export declare namespace Backend {
   /**
    * set options for the WebGL backend
    */
-  interface WebGLOptions {}
+  interface WebGLOptions {
+    /**
+     * set or get the WebGL Context ID (webgl vs webgl2,...)
+     */
+    contextId?: string;
+    /**
+     * set or get the flag which determines which format parameter to pass to gl.readPixels()
+     */
+    forceRgbaReads?: boolean;
```

Comment:
I would prefer to use something like `readPixelsFormat?: 'ALPHA'|'RGB'|'RGBA';`

Reply:
I am removing forceRGBaReads period. which should resolve this issue as well.